en/mozorg/about.ftl

```diff
@@ -37,6 +35,12 @@ about-read-the-manifesto = Read The Manifesto
 about-a-global-view = A Global View
 about-with-offices-all-over-the = With <a href="{ $url }">offices all over the world</a>, we consider the internet from multiple cultures and contexts.
 about-san-francisco = San Francisco
+about-talking-tech-issues-irl = Talking Tech Issues IRL
+about-in-mozillas-multi-award-winning = In { -brand-name-mozilla }'s multi-award winning podcast, host Bridget Todd talks to the people shaping the future of the internet and AI.
+
+# Obsolete strings (remove in 1-2 months)
```

Comment:
@nathan-barrett 

Not blocking on this, but for the future, this should be a group-level comment in Fluent. Right now, it's displayed only for the first string.

```

## Obsolete strings (remove in 1-2 months)

about-talking-internet-issues-irl = Talking Internet Issues IRL
about-in-mozillas-irl-podcast-host = In { -brand-name-mozilla }’s IRL podcast, host Manoush Zomorodi shares real stories of life online and real talk about the future of the Web.

##

```